.github/workflows/cd.yml

```diff
@@ -1,17 +1,65 @@
 name: continuous-delivery
+
 on:
   push:
     branches: [main]
+
```

Comment:
the next two jobs only take care of adding the required libraries for the JS code and pack them all in a single file